student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, _name = 'Unknown', parent_permission: true)
+    super(age, 'Unknown', parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] For the inheritance of the `name` variable, you should put the variable itself in `super`, not the default value of it. Please fix that.
[Hint]: It will be like that:
```ruby
def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission)
    @classroom = classroom
 end
```